### ISSUE TITLE
ESP32-C3 RISC-V single-core cpu

### ISSUE THREAD

**Issue by GlebDen** [bovineck/ESP32-OTA-AP#1]
Hello.

Is it possible to modify you code for ESP32-C3 RISC-V single-core CPU?

**Comment by bovineck** [comment]
Yes, the original code was single core, but I upgraded so one core is available for the html interface and the other for the main project code. Look back through the ChatGPT logs for example. You could copy the whole code to an AI and ask for the revisions you require. Good luck. 👍😀
